samples/sensor/stm32_qdec/README.rst

```diff
@@ -0,0 +1,62 @@
+.. _stm32_qdec_sensor:
+
+STM32 Quadrature Encoder Sensor
```

Comment:
Shouldn't it be rather 'decoder' than 'encoder' ?

Reply:
This is something I also wasn't sure about.
I followed similar drivers' design and since all of them had "qdec" in the name (which I guess stands for "quadrature decoder") I did the same. However ST in their documentation talks about "quadrature encoder" (see RM0368 for example), so I don't know which of the 2 should prevail...

Reply:
Actually this has been solved following a suggestion from @str4t0m. Now the sample application is more generic and should be able to handle also decoders from other manufacturers. As a consequence I changed the name to "decoder"